index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="flex-container">
+           <h1 class="flex-item">Welcome</h1>
+           <a href="#"><img class="flex-item" src="./images/Screenshot_1.jpg" alt=""></a>
+        </nav>
+    </header>
+    <section>
+        <h2>Hey there. I’m Mariane</br>
+            <span class="welcome-text">I’m a software developer</span></h2>
+            <p class="welcome-paragraph"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+    </section>
+    <section class="icon">
+        <div>
+            <img src="./images/twitter.png" alt="twitter">
+        </div>
+        <div>
+            <img src="./images/Linkedin icon.png" alt="Linkedin">
+        </div>
+        <div>
+            <img src="./images/medium.png" alt="Medium">
+        </div>
+        <div>
+            <img src="./images/github.png" alt="Github">
+        </div>
+        <div>
+            <img src="./images/angellist.png" alt="Angellist">
+        </div>
+    </section>
```

Comment:
- [ ] The list of icons shoul be applied using a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Kindly  arrange it. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

Reply:
Fixed